warehouse/query-core/src/main/java/datawave/query/index/lookup/ScannerStream.java

```diff
@@ -1,92 +1,52 @@
 package datawave.query.index.lookup;
 
-import java.util.Collections;
-import java.util.Iterator;
-
+import com.google.common.collect.Iterators;
+import com.google.common.collect.PeekingIterator;
 import datawave.query.jexl.JexlNodeFactory;
-import datawave.query.jexl.visitors.JexlStringBuildingVisitor;
+import datawave.query.tables.RangeStreamScanner;
 import datawave.query.util.Tuple2;
-
-import org.apache.commons.jexl2.parser.ASTReference;
 import org.apache.commons.jexl2.parser.JexlNode;
-import org.apache.commons.jexl2.parser.ParserTreeConstants;
 
-import com.google.common.collect.Iterators;
-import com.google.common.collect.PeekingIterator;
+import java.util.Collections;
+import java.util.Iterator;
 
-public class ScannerStream extends ASTReference implements IndexStream {
-    protected final PeekingIterator<Tuple2<String,IndexInfo>> itr;
-    protected final StreamContext ctx;
-    protected JexlNode currNode;
-    // The debug delegate is for optionally delegating the contextDebug call
-    protected final IndexStream debugDelegate;
+/**
+ * Basic implementation of an IndexStream for a single term.
+ *
+ * Note that certain delayed terms may create a ScannerStream without an underlying RangeStreamScanner.
```

Comment:
I am trying to wrap my head around this one.  Why would this be the case?

Reply:
The comment is wrong, a SHARDS_AND_DAYS hint will generate a list of shards that get intersected. That presents as a scanner stream without a backing RangeStreamScanner. The reference to delayed terms should be removed.

Reply:
Are you going to fix the comment then?

Reply:
Fixed.